inst/scripts/README.md

```diff
@@ -0,0 +1,52 @@
+# Scripts
+
+## update-metadata-template-dictionary.R
+
+This script updates the dictionary sheets in existing metadata template files. A metadata template is expected to be an Excel (.xlsx) file with 3 sheets:
+- template: header of metadata keys
+- dictionary: definitions of the metadata keys present in the template (columns: key, description)
+- values: enumerated values for the metadata keys present in the template (columns: key, value, valueDescription, source)
+
+### Requirements
+
+#### System
+
+This script requires the dccvalidator package (version >= 0.3.0.9012). Install the latest version with:
```

Comment:
Should this be 0.3.0.9011 (the current version after this PR)?

Reply:
The DESCRIPTION actually needs to be updated in this PR. #435 was merged first so this one needs to be bumped to 3.0.9012. I just pushed the change for this.